andela/ah-backend-sparta
# #164798223 The user should able to see their reading stats

#### What does this PR do?
Enables user to view their reading stats

#### Description of Task to be completed?
The endpoint will be able to fetch the most recently reads by the user

#### How should this be manually tested?
- Fetch this branch `git fetch origin ft-see-reading-stats-164798223`
- Checkout to the branch `git checkout ft-see-reading-stats-164798223`
- Run `pipenv shell`
- Run `python manage.py migrate`
- Run `python manage.py runserver`

#### What are the relevant pivotal tracker stories?
 [#164798223](https://www.pivotaltracker.com/story/show/164798223)

#### Screenshots (if appropriate)
<img width="466" alt="Screenshot 2019-04-15 at 16 04 09" src="https://user-images.githubusercontent.com/35653907/56135669-e77ee980-5f99-11e9-9b61-8476075a3662.png">

#### Postman Collection
[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/6d1b648c3e9a9b00e07b)
